templates/android.js

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
I'm a bit concerned that this change doesn't have a lot of to do with the rest of pull request. Am I missing something or could this be delivered separately?

Reply:
yes, resolved this in #59